docs/admin/serverless/serverless.md

```diff
@@ -33,17 +33,26 @@ The minimally required Cert Manager version is 1.15.0 and you can refer to [Cert
     Cert manager is required to provision webhook certs for production grade installation, alternatively you can run self signed certs generation script.
     
 ## 4. Install KServe
-=== "kubectl"
+
+=== "Install using Helm"
     ```bash
-    kubectl apply -f https://github.com/kserve/kserve/releases/download/v{{  kserve_release_version }}/kserve.yaml
+    # Install KServe CRDs
+    helm install kserve-crd oci://ghcr.io/kserve/charts/kserve-crd --version v{{ kserve_release_version }}
+    
+    # Install KServe Resources
+    helm install kserve oci://ghcr.io/kserve/charts/kserve --version v{{ kserve_release_version }}
     ```
-
-## 5. Install KServe Built-in ClusterServingRuntimes
-{{ kserve_release_version }}
-=== "kubectl"
+    
+=== "Install using YAML"
     ```bash
-    kubectl apply -f https://github.com/kserve/kserve/releases/download/v{{ kserve_release_version }}/kserve-cluster-resources.yaml
+    # Install Kserve CRDs and Controller
+    kubectl apply --server-side -f https://github.com/kserve/kserve/releases/download/v{{  kserve_release_version }}/kserve.yaml
+    
+    # Install KServe Built-in ClusterServingRuntimes
+    kubectl apply --server-side -f https://github.com/kserve/kserve/releases/download/v{{ kserve_release_version }}/kserve-cluster-resources.yaml
     ```
+!!! note
+    --server-side option is needed because the InferenceService CRD is too long to process, see [this issue](https://github.com/kserve/kserve/issues/3487) for details.
```

Comment:
This shows for both options, we can move to a comment on the `using YAML` option